git/repo_test.go

```diff
@@ -68,7 +68,7 @@ func TestLog(t *testing.T) {
 	if got, want := patch.Subject, "[PATCH] first commit"; got != want {
 		t.Errorf("got %v, want %v", got, want)
 	}
-	if got, want := patch.Author, `"your name" <you@example.com>`; got != want {
+	if got, want := patch.Author, `your name <you@example.com>`; got != want {
```

Comment:
I don't know enough about grit to predict if this will cause a problem, but I trust that if you tested it and things worked, it's ok.